### DISCUSSION TITLE
Bail Collection/Wrap - How it works?

### DISCUSSION THREAD

**Discussion by FookinNora** [Courseplay/Courseplay_FS22#825] [Q&A] [answered]
Hey,
       So I play on No Man's Land which is a big open mod map that starts with 1 small field and then you basically just create your own. I've run into an issue when using the Bail Collect feature where it just gives the "No Field" message when you start it (through the mini menu with no course loaded and the feature selected as the start point). Every other CP function works when generating field work its all fine, field detected, no issues. I have loaded a base game map and the Bail Collection function works perfectly so no issues with build or method of using it. 
   So I am assuming that the feature uses some sort of pre-existing data about fields from the map its self to function?
I seem to remember having to select a certain field number to make the feature work in FS19 when working with 2 fields that had been combined (via ploughing and the create field feature) but it did work the whole field and not just 1 section meaning it does still use some sort of field detection?
   So I'm guessing that it uses some sort of map data to start its field search? then sets boundaries from there using the same method as the course generation? (Stab in dark *said in that voice where everything is a question :D )  
   If I'm right obviously that's the cause of my problem so was just trying to get confirmation really that that's my issue and obviously be awesome to have it work but I understand this might be a niche issue. And if I'm way off the mark I can go back to testing with some hope :)

**Comment by Tensuko** [comment, marked as answer]
The field where you start on, have to be the field where the bales are.
We already have a request for none fields and combines fields.

**Reply by FookinNora** [reply]
Thanks for the reply man, cool just wanted to double check I wasn't doing something wrong. I will keep my eyes peeled on the new releases
